File: Easy/70-climbing-stairs/70-climbing-stairs.py
class Solution:
    def climbStairs(self, n: int) -> int:
        
        hm = {}
        hm[1]=1
        hm[2] = 2
        for i in range(3, n+1):
            hm[i] = hm[i-1] + hm[i-2]
            
        return hm[n]
        

File: Easy/70-climbing-stairs/70-climbing-stairs.js
/**
 * @param {number} n
 * @return {number}
 */
var climbStairs = function(n) {
    
    let arr={0:0,1:1,2:2}
   
    
    for (let i=3;i<=n;i++){
        arr[i] =arr[i-1]+arr[i-2];
    }
    
    return arr[n];
    
};